crates/viewer/re_viewer_context/src/cache/video_stream_cache.rs

```diff
@@ -316,9 +320,22 @@ fn read_samples_from_chunk(
         .and_then(|time_range| time_range.get(&sample_descr))
     {
         Some(time_range) => {
-            if time_range.min().as_i64() < previous_max_presentation_timestamp.0 {
+            // If the whole chunk is before the current stream time range, insert
+            // it at the start.
+            if time_range.max().as_i64() < min_presentation_timestamp.0 {
+                re_log::warn_once!(
```

Comment:
I think it would be ideal if these warns were "once per recording". Guess I could include the current recording in the warn somehow?